applications/data_analysis/general_applied_math.ipynb

```diff
@@ -0,0 +1,907 @@
+{
```

Comment:
<p>I know that this is (probably) technically accurate, but it feels reductive. I'd prefer if we removed the text under the heading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>